### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by lukman155**
# Portfolio: Create a Mobile Menu

## Learning objectives

- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Understand how to parse a Figma design to create a UI.
- Use objects to store and access data.

## Requirements:
- I used media queries to make sure that my website displays correctly on both mobile and desktop screens.
- I made sure that there are no linter errors.
- I made sure that I used the correct GitHub flow.
- I made sure that I documented my work.
- I documented my work in a professional way.
- I designed the page using responsible design principles.
- I deployed your website using GitHub Pages.
- I checked the online version of my portfolio and made sure that the page works properly.
- I updated the README of my repository to include a link to the online version.
 # 🥇

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+cancel1.addEventListener('click', mobileMenu);
+cancel2.addEventListener('click', mobileMenu);
+
+projectID.addEventListener('mouseover', addLine);
+aboutID.addEventListener('mouseover', addLine1);
+contactID.addEventListener('mouseover', addLine2);
+
+projectID.addEventListener('mouseout', removeLine);
+aboutID.addEventListener('mouseout', removeLine1);
+contactID.addEventListener('mouseout', removeLine2);
```
Comment (mirouhml) [inline script.js]: - [ ] Please know that when I click on `Portfolio` in the mobile menu it doesn't disappear and the X mark disappears, so I have no way of closing the menu.
image

- [ ] Also as you can see in the above screenshot, the menu isn't covering the whole screen you can see under it, please make sure it's covering the whole screen.
- [ ] I suggest that you place the X button inside the mobile menu and not in the header that way it won't disappear even if it's scrolled.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @lukman155, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -15,36 +15,32 @@
...
+                    <span></span>
+                    <span></span>
+                </div>
+
+                <ul class="navi">
+                    <li><a class="color-text cancel" href="#project-section">Portfolio</a></li>
+                    <li><a class="color-text cancel1" href="#about-section">About</a></li>
+                    <li><a class="color-text cancel2" href="#contact-section">Contact</a></li>
+                </ul>
+            </nav>
```
Comment (mukhammadrizooff) [inline index.html:32]: - [x] [REQUIRED] You almost met the requirements mentioned by the previous code reviewer 👍, but it looks like you didn't fully meet them, don't worry, it's very easy to fix them, I trust you to fix them 😉. When the `mobile menu` is `scrolled down`, the open areas at the `top` and `bottom` of the mobile menu container still do not disappear, and even when there is `no scroll`, the body elements are visible at the bottom of the `menu` and the `close button` disappears. Please solve these problems 😃 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @lukman155, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Perfect `close Menu button` animation ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Lukman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**